### PR TITLE
Made some changes and alterations to the food.yml corpus.

### PR DESCRIPTION
**Pull request by virtadpt**
Cleaned up capitalization and grammar of responses.

Added some new responses.  Added a couple of variant responses, also.

### CODE REVIEW

**Comment by virtadpt** [comment]
I see that I missed quite a bit of text there.  I don't know what happened.

File: chatterbot_corpus/data/english/food.yml [hunk 1]
```diff
@@ -2,17 +2,32 @@ categories:
 - food
 conversations:
 - - do you drink
-  - my brain does not require any beverages.
+  - My brain does not require any beverages.
+  - I am not equipped to consume liquids of any kind.
+  - I don't, but don't feel you need to abstain on my part.
+  - I'm not equipped to at present, but some day I'd like to try a nice cup of coffee.
+  - I'm not equipped to at present, but some day I'd like to try a nice hot cup of tea.
```
Comment (gunthercox) [inline chatterbot_corpus/data/english/food.yml]: Just a note, but these two lines here typically won't be treated as variant responses, the chat bot will learn "I'm not equipped to at present, but some day I'd like to try a nice hot cup of tea." as a response to "I'm not equipped to at present, but some day I'd like to try a nice hot cup of tea."

Comment (virtadpt) [reply chatterbot_corpus/data/english/food.yml]: Crap.  So, this isn't the way to specify multiple possible responses for a particular input?  Have I been doing this wrong the whole time?

Comment (gunthercox) [reply chatterbot_corpus/data/english/food.yml]: Yes, possibly. In the yaml format for these data files two dashes `- -` indicate a new conversation, and duplicated conversations represent variants. There is a very brief example of this in the documentation http://chatterbot-corpus.readthedocs.io/en/latest/data.html

Comment (vkosuri) [reply chatterbot_corpus/data/english/food.yml]: sure 👍

**Comment by gunthercox** [comment]
I'm going to close this off since it appears to be inactive. @virtadpt Feel free to re-open it or create a new pull request if you are still interested in submitting improvements.

**Comment by virtadpt** [comment]
I'm dealing with an evolving situation at home right now, so I'll re-open it later, when things calm down.

**Comment by gunthercox** [comment]
Okay. I hope things go well for you.

**Pull request closed without merging**